src/components/InputTodo.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+class InputTodo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      Addtodo: this.props,
+    };
+  }
+
+  onChange = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { title, Addtodo } = this.state;
+    if (title.trim()) {
+      Addtodo.addTodoProps(title);
+      this.setState({ title: '' });
+    } else {
+      alert('Please write an item');
```

Comment:
Remember that alerts, just as console logs, should be avoided in best practice. Once you finish the tutorial, remember to delete or change the alert. For example you could just add a `e.preventDefault()`.